data/stages.lua

```diff
@@ -0,0 +1,30 @@
+experienceStages = {
+	{minlevel = 1, maxlevel = 8, multiplier = 7},
+	{minlevel = 9, maxlevel = 20, multiplier = 6},
+	{minlevel = 21, maxlevel = 50, multiplier = 5},
+	{minlevel = 51, maxlevel = 100, multiplier = 4},
+	{minlevel = 101, multiplier = 3}
```

Comment:
shouldn't variables name follow camel case? am I missing anything?

Reply:
do you talk about the `maxlevel` variable  on the last stage that doesn't exist?

Reply:
Im talking about minlevel, maxlevel in the table, specifically. Not sure what pattern are we following for table 'properties' like table.minlevel. Im used to camelcase here as well (from a javascript world /json patter I'd guess a.minLevel instead). Not sure if the same would apply here. It's more like a general doubt.

Reply:
I don't usually use camelcase when they are short variables, but if the pattern is camelcase, I can change it.

Reply:
I don't really know, it was just a question. Im not the right person to answer that :sweat_smile:

Reply:
Because the var names are both short and the particular letter breaks involved, `nl` `xl`, you can freely get away with using all lowercase here. I'd argue it's even desirable because of the presence of a long var name like multiplier would make capitals in the other two jarring to the eyes.